### DISCUSSION TITLE
Run processes inside container as non-root user

### DISCUSSION THREAD

**Discussion by graduatio** [TandoorRecipes/recipes#1249] [Q&A]
Executing `docker top CONTAINER` shows that the processes inside the recipes container run as user root. From a security perspective it would be better if the processes run as non-root user even if it is inside the container. Is there a way to run the services as a non-root user e.g. by defining a user option in the docker-compose file?

**Comment by smilerz** [comment]
not currently, but probably a good idea - can you submit a feature request?

**Comment by vabene1111** [comment]
we need to pay very close attention when changing this as it can easily break things like the nginx configuration, media files, static files and other stuff.

**Comment by graduatio** [comment]
Thanks for the replies. I will create a feature request.
